### PR TITLE
Updated Blog Header For Dark Mode Compatibility

### LINKED ISSUE #430: Blog logo unreadable when in dark mode
### What happened?

The README logo is unreadable in dark mode, and the borders around the articles also disappears.

Light
image
Dark
image

### PR DESCRIPTION
**Pull request by k4rni**
Converted previous SVG image into a responsive segment of HTML that is better suited for the website and dark mode compatibility.

Mode | Desktop (Light) | Desktop (Dark) | Phone (Light) | Phone (Dark)
---|---|---|---|---
Before  | image | image | image | image
After | image | image | image | image

Fixed #430

### CODE REVIEW

File: src/routes/blog/index.svelte [hunk 1]
```diff
@@ -15,22 +15,31 @@
...
 <svelte:head>
-  <title>acmCSUF / README</title>
+  <title>Blog / ACM at CSUF</title>
 </svelte:head>
 
 <Spacing --min="175px" --med="200px" --max="200px" />
 
-<section>
+<section class="main-header">
   <img src="assets/readme-logomark.svg" alt="README by acmCSUF" />
```
Comment (EthanThatOneKid) [inline src/routes/blog/index.svelte]: I have a feeling that referencing the SVG relatively on the `/blog` page might give us varying results.
```suggestion
  <img src="/assets/readme-logomark.svg" alt="README by acmCSUF" />
```

**Comment by jaasonw** [comment]
umm, where is the blog?


https://user-images.githubusercontent.com/8981287/168467920-47b52104-080b-4f30-83e6-1bf9e592910a.mp4

**Comment by k4rni** [comment]
I'm not sure what happened to the blog but it's not working at all until you refresh it

**Comment by jaasonw** [comment]
it became a 404 after i refreshed it

**Comment by EthanThatOneKid** [comment]
> umm, where is the blog?
> 
>  2022-05-15.03-18-53.mp4

> I'm not sure what happened to the blog but it's not working at all until you refresh it

> it became a 404 after i refreshed it

I suspect our serverless function throws an error when it attempts to read from a JSON file since [Vercel.com](https://vercel.com/support/articles/how-can-i-use-files-in-serverless-functions) currently says, "At present, it is not possible to import files in a Next.js SSR page or function given https://github.com/vercel/next.js/issues/8251." This lead me to create issue #471. 

By eliminating the following line of code, **the blog just might [hopefully] start to work again in preview environments**!
<https://github.com/EthanThatOneKid/acmcsuf.com/blob/486a2ec5251ee9c52807a798eff5899b8b291aeb/src/routes/blog/index.json.ts#L6>
^ I suspect this line of code may be our problem, so I made #472 in an attempt to resolve #471.

File: src/routes/blog/index.svelte [hunk 2]
```diff
@@ -15,22 +15,31 @@
 </script>
 
 <svelte:head>
-  <title>acmCSUF / README</title>
+  <title>Blog / ACM at CSUF</title>
```
Comment (jaasonw) [inline src/routes/blog/index.svelte:18]: Why "Blog" instead of "README"?

File: src/routes/blog/index.svelte [hunk 3]
```diff
@@ -15,22 +15,31 @@
...
+  <div>
+    <h1 class="size-xxl">README</h1>
+    <h2 class="size-md">by acm<b class="acm-blue">CSUF</b></h2>
+  </div>
+</section>
 
-  <h2 class="subtitle headers">
+<section>
+  <h2 class="subtitle headers size-md">
     The official acmCSUF blog.<a href="/blog.xml"
```
Comment (jaasonw) [inline src/routes/blog/index.svelte:34]: acmCSUF -> "ACM at CSUF"

File: src/routes/blog/index.svelte [hunk 4]
```diff
@@ -15,22 +15,31 @@
...
 <Spacing --min="175px" --med="200px" --max="200px" />
 
-<section>
-  <img src="assets/readme-logomark.svg" alt="README by acmCSUF" />
+<section class="main-header">
+  <img src="/assets/readme-logomark.svg" alt="README by acmCSUF" />
+
+  <div>
+    <h1 class="size-xxl">README</h1>
+    <h2 class="size-md">by acm<b class="acm-blue">CSUF</b></h2>
```
Comment (jaasonw) [inline src/routes/blog/index.svelte:28]: acmCSUF -> "ACM at CSUF"

**Review by jaasonw — CHANGES_REQUESTED** [review]

File: src/routes/blog/index.svelte (hunk 2 above)
Comment (k4rni) [reply src/routes/blog/index.svelte:18]: I'll go ahead and change that in the next update

File: src/routes/blog/index.svelte (hunk 4 above)
Comment (k4rni) [reply src/routes/blog/index.svelte:28]: I'll incorporate these changes with #410

**Comment by jaasonw** [comment]
Closing because I merged these changes into https://github.com/EthanThatOneKid/acmcsuf.com/pull/481

File: src/routes/blog/index.svelte (hunk 1 above)
Comment (jaasonw) [reply src/routes/blog/index.svelte]: resolving this cus it seems to work fine

Comment (EthanThatOneKid) [reply src/routes/blog/index.svelte]: Thank you

**Review by jaasonw — APPROVED** [review]

File: src/routes/blog/index.svelte (hunk 2 above)
Comment (jaasonw) [reply src/routes/blog/index.svelte:18]: FIXED IN https://github.com/EthanThatOneKid/acmcsuf.com/pull/481

**Pull request merged**